### PR TITLE
Dialogs & modals

### PR DESCRIPTION
**Pull request by jeetbhalani15**
image

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (rohanmathur91) [inline .vscode/settings.json:3]: you can delete this file or can put in .gitignore

File: components/Badges/badges.css [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    top: -10px;
+    left: 34px;
+    background-color: var(--badge-bg-color);
+    padding: 6px;
+    border-radius: 50%;
+    border: 3px solid white;
+}
+button{
+    padding: 12px 22px;
+}
```
Comment (rohanmathur91) [inline components/Badges/badges.css:54]: you can use rem here.

File: components/Badges/badges.css [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+*/
+.badge__icon{
+    position: absolute;
+    color: var(--badge-text-color);
+    top: -21px;
+    left: 28px;
+    background-color: var(--badge-bg-color);
+    padding: 3px;
+    border-radius: 50%;
+    width: 25px;
```
Comment (rohanmathur91) [inline components/Badges/badges.css:84]: use rem wherever possible

File: components/Buttons/button.css [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+*/
+.lg-btn{
+    font-size: 1.1rem;
+    padding: 0.7rem 1.5rem;
+    min-width: 80px;
+}
+.sm-btn{
+    font-size: 0.9rem;
+    padding: 0.3rem 0.7rem;
+    width: 112px;
```
Comment (rohanmathur91) [inline components/Buttons/button.css:41]: same as above.

File: components/Buttons/button.css [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+}
+/* 
+CSS FOR TOGGLE BTN
+*/
+.btn-toggle{
+    width: 26px;
+    height: 44px;
```
Comment (rohanmathur91) [inline components/Buttons/button.css:48]: same as above.

File: components/Buttons/button.css [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+    border-radius: 50%;
+    background-color: rgb(226, 226, 226);
+    transition: 0.4s;
+}
+.btn-toggle .checkbox:checked + .btn-slider::before{
+    transform: translateX(-18px);
+}
+.btn-toggle .checkbox:checked + .btn-slider{
+    background-color: #66263f;
+}
```
Comment (rohanmathur91) [inline components/Buttons/button.css:86]: have a variable for this color and use that.

File: components/Dialogs/dailog.css [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+ .flex{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+}
+  .box__border{
+    border-radius: 5px;
+    box-shadow: 0px 0px 5px;
+    background-color: rgb(255, 255, 255);
```
Comment (rohanmathur91) [inline components/Dialogs/dailog.css:11]: same as above, use variable

**Review by rohanmathur91 — CHANGES_REQUESTED** [review]
Use rem where value is more than 10px and have variable for all your colors and use that.

**Review by Jay-Gadhiya — APPROVED** [review]

**Review by gopalbbharadva — APPROVED** [review]
Improvement : use relative units instead of absolute unit. (like use rems instead of px). remove unused comments.

**Pull request merged**